### ISSUE TITLE
req.cookies empty on deploy

### ISSUE THREAD

**Issue by markacianfrani** [simov/grant-vercel#1]
My apologies if this is totally unrelated to the package, but I'm running into an issue when trying to deploy to Vercel. I'm getting an error:

```
Unhandled Promise Rejection 	{"errorType":"Runtime.UnhandledPromiseRejection","errorMessage":"TypeError: req.cookies.join is not a function" . . . 
```

req.cookies is an empty object when it gets passed to grant. Specifically I'm trying the transport-querystring example. Everything works perfectly when running `vercel dev`, it's just when I go to deploy which makes me think it's just a dumb config issue. 

This is my first attempt at anything serverless so I'm sure there's huge gaps in my knowledge here.

**Comment by simov** [comment]
Hi, this was a bug in Grant, introduced by Vercel. Last time I checked my test deployment about a year ago it worked. It seems that they have introduced that property long time ago https://vercel.com/blog/vercel-node-helpers so I don't have a good explanation as to how it worked on my end. Maybe I got fooled by having the Grant cookie stored in my browser and so the parser was picking that from the headers object instead of trying the cookies one. Still not quite sure how no one noticed that if that was the case.

In any case, I pushed a fix, but I will need some time to update the tests as well. In the meantime you can use Grant like this in your package.json:

```json
"dependencies": {
  "grant": "simov/grant"
},
```

I'll let you know when the new release is out. Also let me know if you have any other issues.

**Comment by markacianfrani** [comment]
Thanks so much! I was so close too. I landed on that page but must've just glossed over it. 

This project is amazing btw. Working on a side proj and having to setup a whole server just for oauth sucks the soul right out of me.

**Comment by simov** [comment]
The fix was published in v5.4.19
